olp-cpp-sdk-dataservice-read/src/repositories/CatalogRepository.cpp

```diff
@@ -331,21 +331,21 @@ client::CancellationToken CatalogRepository::getLatestCatalogVersion(
 MetadataApi::CatalogVersionResponse CatalogRepository::GetLatestVersion(
     const client::HRN& catalog,
     client::CancellationContext cancellation_context,
-    const DataRequest& data_request, client::OlpClientSettings settings) {
+    const CatalogVersionRequest& request, client::OlpClientSettings settings) {
```

Comment:
Please, pass catalog and request by value, as we already have moves in CatalogClientImpl::GetCatalog